qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -387,11 +451,11 @@ def _build_commutator_routine(
                 # theory, one would choose this according to the nature of the problem (i.e.
                 # whether it is fermionic or bosonic), but in practice, always choosing the
                 # anti-commutator has proven to be more robust.
-                q_mat_op = double_commutator(left_op, operator, right_op_1, sign=False)
+                q_mat_op = -double_commutator(left_op1, operator, right_op_1, sign=False)
                 # In the case of the single commutator, we are always interested in the energy
                 # difference of two states. Thus, regardless of the problem's nature, we will
                 # always use the commutator.
-                w_mat_op = commutator(left_op, right_op_1)
+                w_mat_op = -commutator(left_op1, right_op_1)
```

Comment:
Where do the `-` signs come from?

Reply:
If found inconsistencies in the management of signs so my changes aim at following closely the theoretical formulas of the QEOM paper.
In this case the signs come from Eq(4) of the paper https://arxiv.org/abs/1910.12890. Previously, the expansion operators were built as E and -E^dag instead of E and E^dag, which explains why the sign was not needed

Reply:
Okay thanks for the explanation :+1: